collector/collector.go

```diff
@@ -87,6 +87,12 @@ func (*Exporters) SetConnectionDetails(config SSHConfig, target string) error {
 	return nil
 }
 
+// SetIfaceDescrKeys sets the optional keys in an interface description to include in metrics
```

Comment:
I did this because a) this is the mechanism being used to set the ssh parameters in junos_go and b) junos_go imports config, and config has this variable (declared in interfaces.go) in its scope.